src/extension/extension.ts

```diff
@@ -183,6 +183,7 @@ export async function activate(context: vs.ExtensionContext, isRestart = false)
 	setupLog(config.analyzerLogFile, LogCategory.Analyzer);
 	setupLog(config.flutterDaemonLogFile, LogCategory.FlutterDaemon);
 	setupLog(config.toolingDaemonLogFile, LogCategory.DartToolingDaemon);
+	setupLog(config.dapLogFile, LogCategory.DAP);
```

Comment:
This seems to be reverting some changes I just pushed to `master` to support "dynamic" DAP log file names.